core/tests/phase_select_food_and_climate_test.py

```diff
@@ -9,6 +9,57 @@ def setUp(self):
         self.game = Game(["Smith", "Gould"])
         self.phase = SelectFoodAndClimatePhase(self.game)
```

Comment:
Suggest changing self.phase to self.climate_phase. May be easier to understand immediately.

Reply:
Agree